Makefile

```diff
@@ -33,6 +33,14 @@ ifneq ($(MAKECMDGOALS),$(OSS_HOME)/build-aux/go-version.txt)
   $(info [make] CHART_VERSION=$(CHART_VERSION))
 endif
 
+ifeq ($(SOURCE_DATE_EPOCH)$(shell git status --porcelain),)
```

Comment:
This is worth a comment. I read this as a typo at first.

Reply:
OK